common/datetime.c

```diff
@@ -229,20 +229,24 @@ oe_result_t oe_datetime_now(oe_datetime_t* value)
 {
     oe_result_t result = OE_UNEXPECTED;
     time_t now;
-    struct tm* timeinfo;
+    struct tm timeinfo;
 
     if (value == NULL)
         OE_RAISE(OE_INVALID_PARAMETER);
 
     time(&now);
-    timeinfo = gmtime(&now);
-
-    value->year = (uint32_t)timeinfo->tm_year + 1900;
-    value->month = (uint32_t)timeinfo->tm_mon + 1;
-    value->day = (uint32_t)timeinfo->tm_mday;
-    value->hours = (uint32_t)timeinfo->tm_hour;
-    value->minutes = (uint32_t)timeinfo->tm_min;
-    value->seconds = (uint32_t)timeinfo->tm_sec;
+#ifdef _WIN32
```

Comment:
this file uses ifdef _WIN32
traceh.c instead uses #if defined(__linux__)
Be consistent.  Even better would be to add something like:

#ifdef _WIN32
#define gmtime_r(now, timeinfo) gmtime_s(timeinfo, now)
#endif

to internal/datetime.h and include that from traceh.c (it's already included from this file),
and then just use gmtime_r in both files.

Reply:
Unresolving since not done yet in diffs.